pom.xml

```diff
@@ -238,6 +238,16 @@
     <doxia.version>1.12.0</doxia.version>
   </properties>
 
+  <dependencyManagement>
+    <dependencies>
+      <!-- Upgrade from 2.1.0 to fix https://github.com/checkstyle/checkstyle/issues/14123 -->
+      <dependency>
+        <groupId>org.codehaus.plexus</groupId>
+        <artifactId>plexus-container-default</artifactId>
```

Comment:
too much special attention for doxia-core

can we use exclude ? 
as we do https://github.com/checkstyle/checkstyle/blob/aaead4bfc79ef6a711d83dc6103cbc7d5b7a0e0a/pom.xml#L404-L416
doxia-core is not actual runtime dependency of Checkstyle, but it is leaked to compile and we can not simply remove now

Reply:
i've changed to it use exclusion now:
```
[INFO] --- dependency:3.6.1:tree (default-cli) @ checkstyle ---
[INFO] com.puppycrawl.tools:checkstyle:jar:10.12.7-SNAPSHOT
(..)
[INFO] +- org.apache.maven.doxia:doxia-core:jar:1.12.0:compile
[INFO] |  +- org.apache.maven.doxia:doxia-sink-api:jar:1.12.0:compile
[INFO] |  +- org.apache.maven.doxia:doxia-logging-api:jar:1.12.0:compile
[INFO] |  +- org.codehaus.plexus:plexus-utils:jar:3.3.0:compile
[INFO] |  +- org.codehaus.plexus:plexus-container-default:jar:2.1.0:compile
[INFO] |  |  +- org.codehaus.plexus:plexus-classworlds:jar:2.6.0:compile
[INFO] |  |  \- org.apache.xbean:xbean-reflect:jar:3.7:compile
[INFO] |  +- org.codehaus.plexus:plexus-component-annotations:jar:2.1.0:compile
(..)
```

I did not use this originally because removing transitive doxia dependency could cause arbitrary runtime errors if classes are no longer available.

I assume you know whether CI fully covers the doxia usage or not?

Reply:
We use doxia for generation of website content.
It supposed to be done as standalone tool but we run into bunch of problems and agreed to allow this dependency temporarily to finish project on web site generation.

One day we will remove it as dependency.